app/src/test/java/org/simple/clinic/shortcodesearchresult/ShortCodeSearchResultLogicTest.kt

```diff
@@ -143,6 +143,8 @@ class ShortCodeSearchResultLogicTest {
     val emptyPatientSearchResults = emptyList<PatientSearchResult>()
     whenever(patientRepository.searchByShortCode(shortCode))
         .thenReturn(Observable.just(emptyPatientSearchResults))
+    whenever(bloodPressureDao.patientToFacilityIds(emptyList()))
+        .thenReturn(emptyList())
```

Comment:
Why did we add this?

Reply:
With the current implementation, the effect handler invokes this method to partition the search results as part of the tests. If we don't add this, the tests fail because the stub for this mock will not be present.